Tutorials/whatiswot/13-Thing_Description/13-Thing-Description.vtt

```diff
@@ -0,0 +1,80 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:07.941 
+ In this video, we will provide a general overview of a fundamental concept in the Web of Things —
+
+00:00:07.941 --> 00:00:10.150 
+the Thing Description, or TD.
+
+00:00:10.601 --> 00:00:18.971 
+ A TD acts as a unique blueprint for the respective Thing, offering a standardized way to describe its functionality
+
+00:00:18.971 --> 00:00:21.192 
+and how a Consumer should interact with it.
+
+00:00:21.585 --> 00:00:29.595 
+ A Consumer is any application or service that communicates with a Thing over the network using its Thing Description.
+
+00:00:29.595 --> 00:00:33.453 
```

Comment:
Some other animation to show the Consumer getting the TD and building the requests to the Thing (they should be kept in the video)